docs/home/howto/DATABASES.md

```diff
@@ -297,10 +298,10 @@ replicas: 3
 resources: # Adjust to your configuration, see https://www.pgpool.net/docs/41/en/html/resource-requiremente.html
   limits:
     # cpu: 900m # Set according to your env
-    memory: 176Mi
+    memory: 310Mi
   requests:
     cpu: 250m # Adjust to your env, increase if possible
-    memory: 176Mi
+    memory: 310Mi
 ```
 
 By default, each PgPool pod requires 176 MB of memory. This value has been determined based on
```

Comment:
Change 176 MB to 310 MB

Reply:
```
By default, each PgPool pod requires 176 MB of memory. This value has been determined based on PgPool docs, however after stress testing we need to add several extra megabytes to avoid failed to fork a child issue.
 ```
I assume this sentence is still true.
 I've added a comment in https://github.com/epiphany-platform/epiphany/issues/2561 to update all information about pgpool because its not a scope of this PR.